pkg/scheduler/scheduler.go

```diff
@@ -407,12 +417,6 @@ func (s *Scheduler) processApplicationUpdateEvent(ev *schedulerevent.SchedulerAp
 				acceptedApps = append(acceptedApps, &si.AcceptedApplication{
 					ApplicationID: app.ApplicationID,
 				})
-				// app is accepted by scheduler
-				err = app.HandleApplicationEvent(cache.AcceptApplication)
-				if err != nil {
-					log.Logger().Debug("cache event handling error returned",
-						zap.Error(err))
-				}
```

Comment:
The Accepted state means the app is accepted by the state (as the opposite to Rejected). This is a proper place for transiting the app to `Accepted`. Why do we want to move this to another place?

Reply:
The Accepted state means that the app is ready for scheduling. When we add the app it is not ready for scheduling it it only ready for scheduling when we add an ask to the app.
The application is in a new state until we get that ask added. The application is rejected by the scheduler if it cannot place the app or it already exists etc. So the app exists in a state that is un-schedulable.